### ISSUE TITLE
Pagination For merge-fields

### ISSUE THREAD

**Issue by bassrock** [welpdev/mailchimp-bundle#6]
Currently if you have more than 10 merge fields the merge field synchronization will fail. This is because by default the merge-fields api only returns 10 fields. So when you try and do an update the client will only know about the 10 existing fields instead of how many there really are.

Right now my current solution is to pass a high count number to the merge-fields api.

**Comment by Nightbr** [comment]
Hi,
we can try to handle this by adding a condition if "total_items" from the result > 10.
I add this into ListRepository.php:

```
        # Handle mailchimp default count limit
        if($result['total_items'] > MAILCHIMP_DEFAULT_COUNT_LIMIT){
            $result = $this->mailchimp->get("lists/$listId/merge-fields", array("count" => $result['total_items']));
        }
```

I push to master if you want to test it. 
We will resolve this issue and your other issue and I will make à release.

**Comment by Nightbr** [comment]
I have released the 1.0.8
feel free to reopen this if it is not fixed
